src/local-history.ts

```diff
@@ -0,0 +1,107 @@
+import * as vscode from 'vscode';
+import path = require('path');
+import fs = require('fs');
+
+// export interface FileProperties {
+//     fileName: string;
+//     uri: string;
+//     lastModified: string;
+//     fileHistory?: string[];
+// }
```

Comment:
[minor] the following can be removed at the moment (keep the history of the file clean).